### PR TITLE
test(replay): Create Acceptance tests for Replay Details and List pages

### PR DESCRIPTION
**Pull request by ryan953**
Im running this with ` pytest tests/acceptance/test_replay_detail.py --no-headless=true` and it looks like communicating with snuba is falling apart

Testing Tips: https://develop.sentry.dev/testing/#acceptance-tests

### CODE REVIEW

File: tests/acceptance/test_replay_detail.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+                                        },
+                                    ],
+                                },
+                            ],
+                        },
+                    },
+                    "timestamp": 1661450171405,
+                }
+            ],
+        ]
```
[the remark this answers is not in the document]
Comment (ryan953) [reply tests/acceptance/test_replay_detail.py:242]: yeah, what you see now is just enough to get it working. it's getting refactored and streamlined

**Review by JoshFerge — APPROVED** [review]
🔥

File: tests/acceptance/test_replay_detail.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        self.browser.snapshot("replay detail no feature")
+
+    def test_not_found(self):
+        with self.feature(FEATURE_NAME):
+            slug = f"{self.project.slug}:abcdef"
+            self.path = f"/organizations/{self.org.slug}/replays/{slug}/"
+
+            self.browser.get(self.path)
+            self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+            self.browser.snapshot("replay detail no feature")
```
Comment (billyvg) [inline tests/acceptance/test_replay_detail.py:64]: This should be "not found"

File: tests/acceptance/test_replay_detail.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+        self.login_as(self.user)
+
+        slug = f"{self.project.slug}:{replay_id}"
+        self.path = f"/organizations/{self.org.slug}/replays/{slug}/"
+
+    def test_no_feature(self):
+        self.browser.get(self.path)
+        self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+        self.browser.snapshot("replay detail no feature")
```
Comment (billyvg) [inline tests/acceptance/test_replay_detail.py:55]: We probably don't need this, don't think it's worth making snapshots for

File: tests/acceptance/test_replay_detail.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            self.browser.get(self.path)
+            self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+            self.browser.snapshot("replay detail no feature")
+
+    def test_simple(self):
+        with self.feature(FEATURE_NAME):
+            self.browser.get(self.path)
+            self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+            self.browser.wait_until_not('[data-test-id="loading-placeholder"]')
+            self.browser.snapshot("replay detail")
```
Comment (billyvg) [inline tests/acceptance/test_replay_detail.py:71]: Is this expected to work? It's erroring

File: tests/acceptance/test_replay_list.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            )
+
+        self.login_as(self.user)
+
+        self.path = f"/organizations/{self.org.slug}/replays/"
+
+    def test_no_feature(self):
+        self.browser.get(self.path)
+        self.browser.wait_until_not('[data-test-id="loading-indicator"]')
+        self.browser.snapshot("replay list no feature")
```
Comment (billyvg) [inline tests/acceptance/test_replay_list.py:41]: imo let's remove

**Review by billyvg — COMMENTED** [review]
A data enhancement we could make is to add some URLs so that the URL walker is not always empty

File: tests/acceptance/test_replay_detail.py (hunk 4 above)
Comment (ryan953) [reply tests/acceptance/test_replay_detail.py:71]: ugh, it was working

Comment (ryan953) [reply tests/acceptance/test_replay_detail.py:71]: oh yeah, `replayTimestamps()` isn't working great and returns `NaN`. Needs more mocked data.

I'll also add the missing `.filter()` call to the js code.

**Pull request merged**